tests/thekdar.test.js

```diff
@@ -89,10 +88,65 @@ describe("Test Thekdar", () => {
     thekdar.addWorkerAdress(FORK_ADDRESS, Task.TYPE_FORK);
     const task = new Task();
     task.setType(Task.TYPE_FORK);
-    task.setMessage({});
+    task.setData({});
     const worker = thekdar._getFreeWorker(task);
     expect(worker).toBeInstanceOf(Worker);
     expect(worker).not.toBe(null);
     expect(worker).toHaveProperty("_id");
   });
+  it("should throw error when maximum workers created", () => {
+    const thekdar = new Thekdar();
+    thekdar.addWorkerAdress(FORK_ADDRESS, Task.TYPE_FORK);
```

Comment:
addWorkerAdress? type it seems.

Reply:
yes,,, i dnt knw how test passes :smile: